### DISCUSSION TITLE
Shuffle queue

### DISCUSSION THREAD

**Discussion by avidseeker** [cmus/cmus#1247] [General]
in `man cmus-tutorial`, in step 4:
```
Step 4: Managing The Queue
...
       Note: The queue is not affected by the "shuffle" option described above.
...
```

If so, how can user shuffle queue?

**Comment by gavtroy** [comment]
You can't, although you can open the queue view (4) and run `:rand` to shuffle the order of the tracks currently in the queue. There's also an old wiki page showing how to randomize through an external script; you could wrap this in a loop to shuffle it every 10 seconds or so -- https://github.com/cmus/cmus/wiki/cmus_randomize_queue . Otherwise maybe you can consider adding tracks to a playlist instead of the queue, since playlists can be played in shuffle mode.
